src/app/shared/modules/changelogs/changelog-item/changelog-item.component.ts

```diff
@@ -1,17 +1,36 @@
 import { Component, OnInit, Input } from '@angular/core';
+import { User } from '@entity';
+import { UserService } from 'app/core/user-module/user.service';
 
 @Component({
   selector: 'clark-changelog-item',
   templateUrl: './changelog-item.component.html',
   styleUrls: ['./changelog-item.component.scss']
 })
 export class ChangelogItemComponent implements OnInit {
-
+  user: User;
+  author: any;
   @Input() changelog: any;
+  @Input() contributors: User[];
+
+  constructor(private userService: UserService) {
+  }
+
+  async ngOnInit() {
 
-  constructor() { }
+    if (this.contributors.includes(this.changelog.authorId)) {
+      this.user = await this.userService.getUser(this.changelog.authorId, '');
+      this.author = {
+        name: this.user.name,
+        profileImage: await this.userService.getGravatarImage(this.user.email, 200),
+      };
+    } else {
+      this.author = {
+        name: 'CLARK Editor',
+        profileImage: 'assets/images/logo.png'
+      };
```

Comment:
I noticed in both videos we have CLARK Editor as the user for the changelog, did you test to make sure that a contributor that is the changelog author appears also?

Reply:
<img width="703" alt="image" src="https://github.com/user-attachments/assets/a17a2886-924d-4bdf-a66c-af08230d6036">